group_vars/pas/production.yml

```diff
@@ -0,0 +1,90 @@
+---
+server:
```

Comment:
While you're doing this, do you want to create a `group_vars/pas/common.yml` file for shared settings like hte timezone?